sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -218,14 +219,35 @@ case class CatalogTable(
     tableType: CatalogTableType,
     storage: CatalogStorageFormat,
     schema: Seq[CatalogColumn],
-    partitionColumns: Seq[CatalogColumn] = Seq.empty,
-    sortColumns: Seq[CatalogColumn] = Seq.empty,
+    partitionColumnNames: Seq[String] = Seq.empty,
+    sortColumnNames: Seq[String] = Seq.empty,
+    bucketColumnNames: Seq[String] = Seq.empty,
+    // e.g. (date, country)
+    skewColumnNames: Seq[String] = Seq.empty,
+    // e.g. ('2008-08-08', 'us), ('2009-09-09', 'uk')
+    skewColumnValues: Seq[Seq[String]] = Seq.empty,
     numBuckets: Int = 0,
     createTime: Long = System.currentTimeMillis,
     lastAccessTime: Long = System.currentTimeMillis,
     properties: Map[String, String] = Map.empty,
     viewOriginalText: Option[String] = None,
-    viewText: Option[String] = None) {
+    viewText: Option[String] = None,
+    comment: Option[String] = None) {
```

Comment:
Does this "comment" mean the comment for the table, as defined in the syntax `create table t1 (c1 int) COMMENT 'abc'`?  If it is, shall it be put into `properties`, it seems Hive puts this table comment into TBLPROPERTIES. so it is consistent when creating hive metastore table from CatalogTable object?